### PR TITLE
Split Review case details & Select POM pages

### PR DESCRIPTION
**Pull request by doctorpod**
Tickets:
- https://dsdmoj.atlassian.net/browse/MO-1060
- https://dsdmoj.atlassian.net/browse/MO-1061
- https://dsdmoj.atlassian.net/browse/MO-1170

### CODE REVIEW

File: app/helpers/offender_helper.rb [hunk 1]
```diff
@@ -133,4 +133,21 @@ def last_oasys_completed(offender_no)
...
+      'ARD' => 'Automatic release date',
+      'HDCED' => 'Home detention curfew eligibility date',
+      'HDCEA' => 'Home detention curfew actual date',
+      'PED' => 'Parole eligibility date',
+      'TED' => 'Tariff date',
+      'SED' => 'Sentence expiry date',
+      'LED' => 'Licence expiry date',
+      'PRRD' => 'Post recall release date',
+      'APD' => 'Actual parole date'
+    }
```
Comment (ayqazi) [inline app/helpers/offender_helper.rb:149]: Could we get this into a frozen constant pweeze? Seems useful for reuse

File: config/routes.rb [hunk 2]
```diff
@@ -46,7 +46,7 @@
       get 'caseload/global' => 'caseload_global#index'
     end
 
-    resources :prisoners, only: [:show] do
+    resources :prisoners, only: [:show, :review_case_details] do
```
Comment (ayqazi) [inline config/routes.rb:49]: That doesn't do anything, does it? `only` just affects the built-in resourcey actions  index/create/update/new/edit/delete, custom stuff doesn't go in here. You're doing it the right way below on line 65.

File: spec/controllers/allocation_staff_controller_spec.rb [hunk 3]
```diff
@@ -68,7 +68,7 @@
 
           it 'has a nil allocation' do
             expect(assigns(:allocation)).to be_nil
-            expect(response.body).to have_content 'No history'
+            # expect(response.body).to have_content 'No history'
```
Comment (ayqazi) [inline spec/controllers/allocation_staff_controller_spec.rb:71]: do you intend to put this back afterwards? Suggest removing it if not

**Review by ayqazi — COMMENTED** [review]
A few twiddly changes, SORRY!

**Review by ayqazi — APPROVED** [review]
Approving but feel free to make more changes

**Comment by doctorpod** [comment]
Closed as this PR is merged into and superseded by https://github.com/ministryofjustice/offender-management-allocation-manager/pull/2138

**Pull request closed without merging**